.github/dependabot.yaml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: pip
+    directory: /
+    labels:
+      - enhancement
+    schedule:
+      interval: weekly
+    ignore:
+      - dependency-name: "galaxy-importer"
```

Comment:
ignored this one because I guess we should just be aligned with what is [on ansible-galaxy side](https://github.com/ansible/galaxy/blob/devel/requirements/requirements.in#L5)